dashed/assets/javascripts/modules/dashed.js

```diff
@@ -169,16 +169,52 @@ var px = (function () {
         }
         return qrystr;
       },
+      widgetHeader: function () {
```

Comment:
yeah it would return an empty selector, maybe we can sort this out later if needed, I'm changing to widgetHeader and missing python's `@property` decorator...